### PR TITLE
Run as a non-root user

### PR DESCRIPTION
**Pull request by arthurbarr**

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -66,20 +66,20 @@ The following is an *example* `Dockerfile` for creating your own pre-configured
...
+Here is an example corresponding `20-config.mqsc` script, which creates two local queues:
 
 ```mqsc
-DEFINE CHANNEL(PASSWORD.SVRCONN) CHLTYPE(SVRCONN) REPLACE
-SET CHLAUTH(PASSWORD.SVRCONN) TYPE(BLOCKUSER) USERLIST('nobody') DESCR('Allow privileged users on this channel')
-SET CHLAUTH('*') TYPE(ADDRESSMAP) ADDRESS('*') USERSRC(NOACCESS) DESCR('BackStop rule')
-SET CHLAUTH(PASSWORD.SVRCONN) TYPE(ADDRESSMAP) ADDRESS('*') USERSRC(CHANNEL) CHCKCLNT(REQUIRED)
-ALTER AUTHINFO(SYSTEM.DEFAULT.AUTHINFO.IDPWOS) AUTHTYPE(IDPWOS) ADOPTCTX(YES)
-REFRESH SECURITY TYPE(CONNAUTH)
+DEFINE QLOCAL(MY.QUEUE.1) REPLACE
```
Comment (parrobe) [inline docs/usage.md]: Why did you delete the MQSC here?

File: install-mq.sh [hunk 2]
```diff
@@ -102,10 +104,8 @@ $UBUNTU && apt-get purge -y \
 $UBUNTU && apt-get autoremove -y
 
 # Recommended: Create the mqm user ID with a fixed UID and group, so that the file permissions work between different images
-$UBUNTU && groupadd --system --gid 999 mqm
-$UBUNTU && useradd --system --uid 999 --gid mqm mqm
-$RHEL && groupadd --system --gid 888 mqm
-$RHEL && useradd --system --uid 888 --gid mqm mqm
+groupadd --system --gid ${mqm_uid} mqm
+useradd --system --uid ${mqm_uid} --gid mqm --groups 0 mqm
 usermod -aG mqm root
```
Comment (parrobe) [inline install-mq.sh]: If we're running as mqm now then i don't think this line is required anymore.

File: internal/containerruntime/runtime.go [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+func GetBaseImage() (string, error) {
+	buf, err := ioutil.ReadFile("/etc/os-release")
+	if err != nil {
+		return "", fmt.Errorf("Failed to read /etc/os-release: %v", err)
+	}
+	lines := strings.Split(string(buf), "\n")
+	for _, l := range lines {
+		if strings.HasPrefix(l, "PRETTY_NAME=") {
```
Comment (parrobe) [inline internal/containerruntime/runtime.go]: Do we want a fall back if we can't find a pretty name variable?

File: docs/usage.md (hunk 1 above)
Comment (arthurbarr) [reply docs/usage.md]: Because it doesn't actually work with the Developer image.  I discovered this while making changes for the users.

File: internal/containerruntime/runtime.go (hunk 3 above)
Comment (arthurbarr) [reply internal/containerruntime/runtime.go]: Could do, but it's not an issue on any of the ones we support

**Comment by arthurbarr** [comment]
Related to #197
